src/lib.rs

```diff
@@ -6,12 +6,10 @@
 //! Using PKCS1v15.
 //! ```
 //! use rsa::{PublicKey, RsaPrivateKey, RsaPublicKey, PaddingScheme};
-//! # /*
-//! use rand::rngs::OsRng;
+//! use rand_core::OsRng;
```

Comment:
I think it's helpful to re-export `rand_core`. 

It'd also be nice if there were a cargo feature to enable `getrandom` which toggles on `rand_core/getrandom`.

If there is a proper feature, we could also add helper methods which use `OsRng` automatically. Looking at the rest of the code, it looks like there are some places where that's already occurring and others where it's parameterized.

Reply:
I am not sure whether we should use `OsRng` as a default RNG. `ThreadRng` looks like a better choice. We could add `rand` as an optional (but enabled by default) dependency and gate on it convenience methods which use `ThreadRng`.